openstack/sap-seeds/templates/flavor-seed.yaml

```diff
@@ -621,3 +621,13 @@ spec:
       "trait:CUSTOM_NUMASIZE_C48_M729": "required"
       "hw:cpu_cores": "3"
       "vmware:hw_version": "vmx-18"
+  # HANA flavor for testing
+  - name: "hana_cc3test"
+    id: "21"
```

Comment:
Please give this flavor a new id from the hana range ("312" seems to be the next free)

Reply:
set flavor id to 312